File: src/main/java/gg/mineral/practice/commands/settings/ToggleDuelRequestsCommand.java
package gg.mineral.practice.commands.settings;

import gg.mineral.practice.commands.PlayerCommand;

import gg.mineral.practice.managers.ProfileManager;
import gg.mineral.practice.util.messages.impl.ChatMessages;
import lombok.val;

public class ToggleDuelRequestsCommand extends PlayerCommand {

	public ToggleDuelRequestsCommand() {
		super("toggleduelrequests");
	}

	@Override
	public void execute(org.bukkit.entity.Player pl, String[] args) {
		val profile = ProfileManager.getOrCreateProfile(pl);
		profile.getRequestHandler().setDuelRequests(!profile.getRequestHandler().isDuelRequests());
		ChatMessages.DUEL_REQUESTS_TOGGLED.clone()
				.replace("%toggled%", profile.getRequestHandler().isDuelRequests() ? "enabled" : "disabled")
				.send(pl);
	}
}

File: src/main/kotlin/gg/mineral/practice/commands/settings/ToggleDuelRequestsCommand.kt
package gg.mineral.practice.commands.settings

import dev.rollczi.litecommands.annotations.command.Command
import dev.rollczi.litecommands.annotations.context.Context
import dev.rollczi.litecommands.annotations.execute.Execute
import gg.mineral.practice.entity.Profile
import gg.mineral.practice.util.messages.impl.ChatMessages

@Command(name = "toggleduelrequests")
class ToggleDuelRequestsCommand {
    @Execute
    fun execute(@Context profile: Profile) {
        profile.duelRequests = !profile.duelRequests
        profile.message(
            ChatMessages.DUEL_REQUESTS_TOGGLED.clone()
                .replace("%toggled%", if (profile.duelRequests) "enabled" else "disabled")
        )
    }
}
